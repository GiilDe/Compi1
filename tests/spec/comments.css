/*simple comment #1*/
/*
comment
multiple 
lines
*/
/*

*/
/*
/////
*****
*/
/***********/
/*
	
///// *
*/
/*

/**/*/