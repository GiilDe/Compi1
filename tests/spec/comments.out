1 COMMENT 1
6 COMMENT 5
9 COMMENT 3
13 COMMENT 4
14 COMMENT 1
18 COMMENT 4
Warning nested comment
